src/Domain.jl

```diff
@@ -1503,6 +1503,14 @@ end
             dydt[d.indexes[3]] -= inter.Vout(t)
         end
     end
+    for inter in interfaces
+        if isa(inter,ConstantVOutlet) && d == inter.domain #ConstantVOutlet has to be the last to be evaluated to work correctly
```

Comment:
It doesn't actually have to be evaluated last. It just has to be evaluated after dVdt has been modified by anything else. This distinction is super important for anyone adding a similar interface such as one that adds gas to keep the pressure constant.

Reply:
Yep you're right